### PR TITLE
RHBPMS-965 - Fix missing i18n

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-properties-editor/uberfire-widgets-properties-editor-client/src/main/java/org/uberfire/ext/properties/editor/client/resources/i18n/CommonConstants.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
```
Comment (manstis) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-properties-editor/uberfire-widgets-properties-editor-client/src/main/java/org/uberfire/ext/properties/editor/client/resources/i18n/CommonConstants.java]: License looks wrong... copy and pasted by the look of it (double asterisk).

Comment (cristianonicolai) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-properties-editor/uberfire-widgets-properties-editor-client/src/main/java/org/uberfire/ext/properties/editor/client/resources/i18n/CommonConstants.java]: fixed, thanks.

**Comment by manstis** [comment]
+1 to merge.

**Comment by ederign** [comment]
Thanks! /cc @paulovmr

**Pull request merged**